### ISSUE TITLE
Rounding numeric values

### ISSUE THREAD

**Issue by Alex-Sokolov** [RobinHerbots/Inputmask#1540]
Using:
```
$('#test').inputmask({
  alias: 'numeric',
  rightAlign: false,
  digits: 0
});
```
When pasting value `30101810100000000602` on blur it will be changed to `30101810100000002000`

[Demo](http://jsfiddle.net/qpngr09j/1/)

Using inputmask 3.3.5-34

**Comment by HenryP3** [comment]
Hi @Alex-Sokolov, I had the same issue as you, I forked and made some changes, there's a pull request with the fix, you can check it out here [#1542](https://github.com/RobinHerbots/Inputmask/pull/1542), but you must not use the minus sign

**Comment by RobinHerbots** [comment]
@Alex-Sokolov , @HenryP3 ,
This issues is solved by the refactoring of the numeric extension.
